aten/src/ATen/mps/MPSDevice.h

```diff
@@ -66,7 +66,8 @@ class TORCH_API MPSDevice {
    */
   bool isMacOS13Plus(MacOSVersion version) const;
 
-  MTLFunction_t metalIndexingFunction(const std::string &kernel, MTLFunctionConstantValues_t constantValues);
+  MTLComputePipelineState_t metalIndexingFunction(const std::string &kernel);
```

Comment:
Hi @DenisVieriu97, do you think it would be better to rename it as `metalIndexingPSO`? If so, I can propose a PR for it.

Reply:
@qqaatw this sounds good! Please go ahead with the change

Reply:
@DenisVieriu97 here is the PR :) #101156